test/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: "../.eslintrc.js",
```

Comment:
I don't think you need this, eslint extends the parent config by default

Reply:
Cool, done